### ISSUE TITLE
BT keyboard writes to console

### ISSUE THREAD

**Issue by spacefrogg** [radekp/qtmoko#116]
After successfully connecting a bluetooth keyboard to the GTA04 all keyboard input seems to be sent to the system console 'behind' the X session. 
It seems that it must be claimed by the neokbdhandler to function properly.

**Comment by radekp** [comment]
Yup exactly. The input method system should be quite well documented:

http://radekp.github.com/qtmoko/api/qt-embedded-charinput.html
http://radekp.github.com/qtmoko/api/tut-deviceexample.html

It should not be that hard to make it working at this point.
